packages/pds/src/services/actor.ts

```diff
@@ -244,6 +244,26 @@ export class ActorService {
       keyset,
     }).execute()
   }
+
+  async deleteUser(did: string): Promise<void> {
+    this.db.assertTransaction()
+    await Promise.all([
```

Comment:
I wonder if performing these three deletes serially might be more reliable for us.  In particular, I think we have a race between the subquery in the `user` deletion and the `did_handle` deletion.

Reply:
yup great catch 🙏